sdks/tool-sdk/src/main/java/org/jreleaser/sdk/tool/AbstractTool.java

```diff
@@ -32,12 +32,14 @@ public class AbstractTool {
     protected final JReleaserContext context;
     protected final DownloadableTool tool;
     protected final String name;
+    protected final boolean verifyErrorOutput;
 
-    public AbstractTool(JReleaserContext context, String name, String version) {
+    public AbstractTool(JReleaserContext context, String name, String version, boolean verifyErrorOutput) {
```

Comment:
Please add an overloaded constructor with the last boolean flag set to `false` as default. This would allow the `Upx.java` tool to remain unchanged.

Reply:
Done :)